package.json

```diff
@@ -1,23 +1,23 @@
 {
-  "name": "react-file-viewer",
+  "name": "@trussworks/react-file-viewer",
   "version": "1.2.1",
   "description": "Extendable file viewer for web",
   "main": "dist/index.js",
   "module": "dist/index.js",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/plangrid/react-file-viewer.git"
+    "url": "git+https://github.com/trussworks/react-file-viewer.git"
   },
   "scripts": {
     "dev": "webpack -d --watch",
     "build": "webpack -p --progress",
     "start": "node ./scripts/start.js",
-    "lint": "node_modules/.bin/eslint ./src/**/*.jsx",
-    "test": "node_modules/.bin/jest --env=jsdom --watch",
-    "jest": "node_modules/.bin/jest --env=jsdom",
+    "lint": "eslint --ext js,jsx src",
+    "test": "jest --env=jsdom ",
+    "test:watch": "jest --watch --env=jsdom",
     "tag-and-publish": "node ./scripts/publish.js"
   },
-  "author": "PlanGrid <opensource@plangrid.com>",
+  "author": "Suzanne Rozier <suz@truss.works>",
```

Comment:
updated so that all are only included in `contributors` (which is consistent with what's in @trussworks/react-uswds). I don't know if we have a generic open source trussworks email.